kits/arm/ar_control_sm.py

```diff
@@ -159,11 +159,9 @@ def compute_arm_goal(self, arm_inputs: ArmMobileIOInputs):
 
 
 def setup_mobile_io(m: 'MobileIO'):
-    m.set_button_label(1, '⟲')
-    m.set_button_label(5, 'arm')
-    m.set_button_mode(5, 1)
-    m.set_button_label(7, 'grip')
-    m.set_button_mode(7, 1)
+    # Set buttons to toggle
+    m.set_button_mode(2, 1)
+    m.set_button_mode(3, 1)
```

Comment:
Can these be set from the layout file?  (In a separate layout file PR, but good to do there when you do)